test/functional/wallet_spend_unconfirmed.py

```diff
@@ -0,0 +1,297 @@
+#!/usr/bin/env python3
```

Comment:
```bash
File "test/functional/wallet_spend_unconfirmed.py" contains a shebang line, but has the file permission 644 instead of the expected executable permission 755. Do "chmod 755 test/functional/wallet_spend_unconfirmed.py" (or remove the shebang line).
ERROR: There were 1 failed tests in the lint-files.py lint test. Please resolve the above errors.
```

This also needs to be added to the list of tests in `test_runner.py`. Which should deal with:
```bash
�[1mWARNING!�[0m The following scripts are not being run: ['wallet_spend_unconfirmed.py']. Check the test lists in test_runner.py.
```